aws/data_source_aws_instance_test.go

```diff
@@ -22,6 +22,7 @@ func TestAccAWSInstanceDataSource_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("data.aws_instance.web-instance", "tags.%", "1"),
 					resource.TestCheckResourceAttr("data.aws_instance.web-instance", "instance_type", "m1.small"),
 					resource.TestMatchResourceAttr("data.aws_instance.web-instance", "arn", regexp.MustCompile(`^arn:[^:]+:ec2:[^:]+:\d{12}:instance/i-.+`)),
+					resource.TestCheckNoResourceAttr("data.aws_instance.web-instance", "user_data_base64"),
```

Comment:
Does it makes sense to add a test case for the case where `get_user_data = true`? I would expect the results to be the same since there is no actual user data provided.

Reply:
Added covering test 👍 

```
--- PASS: TestAccAWSInstanceDataSource_GetUserData_NoUserData (210.42s)
```